backend/src/api/petition-signature/content-types/petition-signature/schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "collectionType",
+  "collectionName": "petition_signatures",
+  "info": {
+    "singularName": "petition-signature",
+    "pluralName": "petition-signatures",
+    "displayName": "Petition Signature",
```

Comment:
We hanteren normaliter Nederlandse termen voor de Display name. Dus... "Petitie-ondertekening"?

Reply:
Of gewoon Petitie-ondertekeningen?